react-common/components/share/ShareInfo.tsx

```diff
@@ -153,124 +142,139 @@ export const ShareInfo = (props: ShareInfoProps) => {
-                        <div className="project-share-text">
-                            {lf("Your project is ready! Use the address below to share your projects.")}
-                        </div>
-                        <div className="common-input-attached-button">
-                            <Input
-                                handleInputRef={handleInputRef}
-                                initialValue={shareData.url}
-                                readOnly={true}
-                                onChange={setName} />
-                            <Button className={copySuccessful ? "green" : "primary"}
-                                title={lf("Copy link")}
-                                label={copySuccessful ? lf("Copied!") : lf("Copy link")}
-                                leftIcon="fas fa-link"
-                                onClick={handleCopyClick}
-                                onBlur={handleCopyBlur} />
-                        </div>
-                        <div className="project-share-actions">
-                            <Button className="circle-button gray embed mobile-portrait-hidden"
-                                title={lf("Show embed code")}
-                                leftIcon="fas fa-code"
-                                onClick={handleEmbedClick} />
-                            <SocialButton className="circle-button facebook"
-                                url={shareData?.url}
-                                type='facebook'
-                                heading={lf("Share on Facebook")} />
-                            <SocialButton className="circle-button twitter"
-                                url={shareData?.url}
-                                type='twitter'
-                                heading={lf("Share on Twitter")} />
-                            {navigator.share && <Button className="circle-button device-share"
-                                title={lf("Show device share options")}
-                                ariaLabel={lf("Show device share options")}
-                                leftIcon={"icon share"}
-                                onClick={handleDeviceShareClick}
+                </div>
+            }
+            <div className="project-share-content">
+                {(prePublish|| shareState === "publish") && <>
+                    <div className="project-share-title">
+                        <h2>{inputTitle}</h2>
+                    </div>
+                    {showDescription && <>
+                        <Input
+                            className="name-input"
+                            initialValue={name}
+                            placeholder={lf("Name your project")}
+                            onChange={setName} />
+                        {isLoggedIn && <Checkbox
+                            id="persistent-share-checkbox"
+                            label={lf("Allow people to see future changes to my project")}
```

Comment:
```suggestion
                            label={lf("Allow people to see future changes to my project.")}
```

Reply:
Just a nit. Wasn't sure which is preferred but based this on other strings in the file which seem to include final period punctuation.